table/block_fetcher.h

```diff
@@ -64,6 +64,8 @@ class BlockFetcher {
         memory_allocator_compressed_(memory_allocator_compressed),
         for_compaction_(for_compaction) {}
 
+  ~BlockFetcher() { status_.PermitUncheckedError(); }
```

Comment:
Do the tests pass if the PermitUncheckedError is done in the constructor?

Reply:
Fixed.